README.md

```diff
@@ -1,59 +1,151 @@
-# Node.js Client for Google Cloud Speech API ([Beta](https://github.com/GoogleCloudPlatform/google-cloud-node#versioning))
+<img src="https://avatars2.githubusercontent.com/u/2810941?v=3&s=96" alt="Google Cloud Platform logo" title="Google Cloud Platform" align="right" height="96" width="96"/>
 
-[Google Cloud Speech API][Product Documentation]: Google Cloud Speech API.
-- [Client Library Documentation][]
-- [Product Documentation][]
+# [Google Cloud Speech API: Node.js Client](https://github.com/GoogleCloudPlatform/google-cloud-node)
 
-## Quick Start
-In order to use this library, you first need to go through the following steps:
+[![release level](https://img.shields.io/badge/release%20level-beta-yellow.svg?style&#x3D;flat)](https://cloud.google.com/terms/launch-stages)
+[![CircleCI](https://img.shields.io/circleci/project/github/GoogleCloudPlatform/google-cloud-node.svg?style=flat)](https://circleci.com/gh/GoogleCloudPlatform/google-cloud-node)
+[![AppVeyor](https://ci.appveyor.com/api/projects/status/github/GoogleCloudPlatform/google-cloud-node?branch=master&svg=true)](https://ci.appveyor.com/project/GoogleCloudPlatform/google-cloud-node)
+[![codecov](https://img.shields.io/codecov/c/github/GoogleCloudPlatform/google-cloud-node/master.svg?style=flat)](https://codecov.io/gh/GoogleCloudPlatform/google-cloud-node)
 
-1. [Select or create a Cloud Platform project.](https://console.cloud.google.com/project)
-2. [Enable the Google Cloud Speech API.](https://console.cloud.google.com/apis/api/speech)
-3. [Setup Authentication.](https://googlecloudplatform.github.io/google-cloud-node/#/docs/google-cloud/master/guides/authentication)
+> Node.js idiomatic client for [Speech API][product-docs].
 
-### Installation
-```
-$ npm install --save @google-cloud/speech
-```
+The [Cloud Speech API](https://cloud.google.com/speech/docs) enables easy integration of Google speech recognition technologies into developer applications. Send audio and receive a text transcription from the Cloud Speech API service.
+
+
+* [Speech API Node.js Client API Reference][client-docs]
+* [github.com/GoogleCloudPlatform/google-cloud-node](https://github.com/GoogleCloudPlatform/google-cloud-node)
```

Comment:
Probably meant to be a link to this repo.

Reply:
Fixed